sale_order_price_recalculation/README.rst

```diff
@@ -0,0 +1,60 @@
+===================================================
+Recalculation of sales order lines prices on demand
+===================================================
+
```

Comment:
Can you add the licence badge. IMO, we should move the licence badge under the module title otherwise it's not displayed on the app store since when the file is converted to html the badge is outside a section element. I'll propose a PR on the template in maintainer-tools to move the badge under the module title.